packages/cursorless-engine/src/processTargets/modifiers/scopeHandlers/NotebookCellScopeHandler.ts

```diff
@@ -9,35 +9,46 @@ import {
 import type { LanguageDefinitions } from "../../../languages/LanguageDefinitions";
 import { ide } from "../../../singletons/ide.singleton";
 import { NotebookCellTarget } from "../../targets";
+import { BaseScopeHandler } from "./BaseScopeHandler";
 import type { TargetScope } from "./scope.types";
 import type {
+  ComplexScopeType,
   ScopeHandler,
   ScopeIteratorRequirements,
 } from "./scopeHandler.types";
 
-export class NotebookCellScopeHandler implements ScopeHandler {
+export class NotebookCellScopeHandler extends BaseScopeHandler {
```

Comment:
The problem was that we were not using the base scope handler and without it we don't always have a distal position which broke our query search range logic.